app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TweetController < ApplicationController
+
+	get '/tweets/new' do 
+		if logged_in?
+		erb :'/tweets/create_tweet'
+	else
+		redirect to '/login'
+	end
+	end
+
+  post '/tweets' do
+  	if params[:content] != ""
+	  	@tweet = Tweet.new(params)
+	  	@tweet.user = current_user
+	  	@tweet.save
+	  	redirect '/tweets'
+	  end
+	  redirect '/tweets/new'
+  end
+
+
+	get '/tweets' do 
+		if logged_in?
+		@tweets = Tweet.all
+		erb :'/tweets/tweets'
+	else
+		redirect to '/login'
+	end
+	end
+
+  get '/tweets/:id' do
+  	if logged_in?
+	  	@tweet = Tweet.find_by_id(params[:id])
+	  	erb :'tweets/show_tweet'
+	  else
+	  	redirect '/login'
+	  end
+  end
+
+	get '/tweets/:id/edit' do 
+		if logged_in?
+		@tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?